app/views/application_templates/show.haml

```diff
@@ -2,6 +2,21 @@
   The #{@template.position.name_and_department} application
   is not currently available.
 - else
+  - if @current_user.try :staff?
+    - if @template.active?
+      Not hiring currently? Click here to take down the application:
```

Comment:
"Click here" vs "Click below"

Also, tone here is a little different than the rest of the site (not that I mind, per se)